libs/island-ui/core/src/lib/Table/Table.tsx

```diff
@@ -19,11 +19,13 @@ type DataField = {
 type Table = {
   children?: ReactNode
   box?: Omit<UseBoxStylesProps, 'component'>
+  className?: string
 }
 
 export const Table = ({
   children,
   box,
+  className,
```

Comment:
We'd like to avoid the className prop as much as possible in `island-ui/core` as it becomes much harder to maintain. Rather add explicit features in the components using typed props.